src/HPWHHeatSources.cc

```diff
@@ -38,15 +40,15 @@ HPWH::HeatSource::HeatSource(HPWH* parentInput, const std::shared_ptr<Courier::C
 {
 }
 
-HPWH::HeatSource::HeatSource(const HeatSource& hSource):Sender(hSource) { *this = hSource; }
+HPWH::HeatSource::HeatSource(const HeatSource& hSource) : Sender(hSource) { *this = hSource; }
 
 HPWH::HeatSource& HPWH::HeatSource::operator=(const HeatSource& hSource)
 {
     if (this == &hSource)
     {
         return *this;
     }
-    Sender::operator = (hSource);
+    Sender::operator=(hSource);
```

Comment:
TODO: Check if this is necessary.

Reply:
This seems to be needed to copy the courier from hSource. (Could use `courier = hSource.courier`..., instead.)

Reply:
Needed because the HeatSource copy constructor is used to assign configured heat sources from local variables into the heatSources vector member of HPWH. Sender data is not transferred implicitly.

Reply:
Makes sense! I think Sender data is transferred implicitly in the implicit assignment operator, but since this is an overriding assignment operator, we need to explicitly call the base class operator function.